assets/src/__tests__/util/math.test.js

```diff
@@ -0,0 +1,7 @@
+import { average } from '../../util/math'
+
+describe('average', () => {
+  it('takes an array of numbers and calculates average', () => {
+    expect(average([1,2,3,4,5])).toEqual(3)
+  })
+})
```

Comment:
It's better to create another pull request for the unit test for utility functions I think?

Reply:
I think it is fine to have all test in one PR, it doesn't hurt that way.

Reply:
gotcha